docs/Chef-CLI-Catalog.md

```diff
@@ -0,0 +1,91 @@
+# Chef CLI (Catalog)
+
+Throughout the history of Chef, we have created and acquired a variety
+of tools that have grown to a point that, as a Chef practitioner, it is
+hard to know, discover and understand all of them. A few of the tools
+that our users use (almost) every day are:
+
+* chef (CLI that includes Policyfiles management)
+* hab (Habitat CLI)
+* inspec
+* ohai
+* kitchen (Test-kitchen)
+* knife
+* berks (Berkshelf)
+* cookstyle
+* delivery
+* chef-apply
+* chef-client
+* chef-run
+* chef-shell
+* chef-solo
+
+## Goals
+* Have a single, unified way to discover tools to interact with Chef products
+* Huge speed improvements
+* Gather data about how our users consume and interact with our tools (Telemetry)
+* Standardize the UX of our (CLI) tools
+* Have the flexibility to rename tools and commands within our ecosystem
+* Enable developers to use any supported programming languages at Chef for efficiency
+
+## Motivation
+
+    As a Chef practitioner,
+    I would like to have a single Command Line Interpreter (CLI) that lets me easily interact with every Chef product.
+    I would also like to have a way to discover new tools that can help me with my day to day work.
+    Finally, since I run these tools daily, I need them to run fast
+
+    As a Chef developer,
+    I would have a tremendous benefit from this architecture since
```

Comment:
These I woulds are future tense assuming this has been implemented. I think they need to be written as what is desired. It's also pre-judging the architecture before garnering feedback. I'd suggest more along the lines of: I would like a more flexible architecture that allows me to invest in improvements selectively where they will have the most impact.